plugins/lookup/lookup.py

```diff
@@ -195,6 +195,7 @@ def get_endpoint(nautobot, term):
         "device-types": {"endpoint": nautobot.dcim.device_types},
         "device-redundancy-groups": {"endpoint": nautobot.dcim.device_redundancy_groups},
         "devices": {"endpoint": nautobot.dcim.devices},
+        "dynamic-groups": {"endpoint": nautobot.extras.dynamic_groups},
```

Comment:
We really need to shore up this list with the list of modules, but every module that we add I want to have the corresponding lookup endpoint (unless it's not applicable for some reason).

Reply:
Ok, the static_group_associations are a many to many endpoint.  I do not think that is applicable here?

Reply:
If there is a GET call and you can send in a filter and it returns items, it should apply. This just hard-codes the respective `app` for the `model` so you can use `static-group-associations` in the lookup plugin:

```yml
    - set_fact:
        group_association: "{{ lookup('networktocode.nautobot.lookup', 'static-group-associations', api_endpoint=nautobot_url, token=nautobot_token, api_filter='foo=\"bar\"') }}"
```

I'm not sure if anyone would actually use it, but since it won't work without the app to model mapping let's just future proof it.

Reply:
Yeah I understand that piece, but I just doubted anybody would actually use it.  Adding it.